src/frontend/src/styles/pages/MyGroup.scss

```diff
@@ -66,3 +66,7 @@
 .project:hover {
     filter: grayscale(0);
 }
+
+.cds--actions-list{
+    column-gap: 4px !important;
+}
```

Comment:
This can shift to the `_overrides.scss` file. Then we have a single source of truth for the override, don't need the `!important` and can fix it globally 😉